src/components/ContactCard/ContactForm/formValuesToContact.spec.js

```diff
@@ -66,6 +66,11 @@ describe('formValuesToContact', () => {
         groups: {
           data: []
         }
+      },
+      displayName: 'John Doe',
+      indexes: {
+        byFamilyNameGivenNameEmailCozyUrl:
+          'DoeJohnjohn.doe@cozycloud.cchttps://johndoe.mycozy.cloud'
```

Comment:
I'm not sure that this is what we want at the end... 

We should probably remove the `http(s)://` from the cozy-url.